influxdb3/src/commands/serve.rs

```diff
@@ -372,6 +377,30 @@ pub async fn command(config: Config) -> Result<()> {
     Ok(())
 }
 
+async fn setup_telemetry_store(
+    object_store_config: &ObjectStoreConfig,
+    instance_id: Arc<str>,
+    num_cpus: usize,
+) -> Arc<TelemetryStore> {
+    let os = std::env::consts::OS;
+    let influxdb_pkg_version = env!("CARGO_PKG_VERSION");
+    let influxdb_pkg_name = env!("CARGO_PKG_NAME");
+    let influx_version = format!("{}-{}", influxdb_pkg_name, influxdb_pkg_version);
```

Comment:
Can you add a comment with some examples of what you expect this to be? e.g. "influxdb3-3.0.1"